cli/Makefile

```diff
@@ -66,7 +66,11 @@ fmt-go: $(GO_FILES) go.mod go.sum
 install: | ./package.json
 	pnpm install --filter=cli
 
-e2e: install turbo
+corepack:
+	npm install -g corepack@latest
+	corepack enable
+
+e2e: corepack install turbo
```

Comment:
i'm sure i missed this in a slack thread somewhere, but is this essentially `pnpm install turbo`  ?

Reply:
Nope! It reads hilariously close to that; but it's actually the enumeration of tasks that need to be run in advance of running the `e2e` task.

Reply:
oh... TIL. this is how Makefiles declare dependencies?